xml/System.Numerics/Complex.xml

```diff
@@ -1635,31 +1635,6 @@ The addition of a complex number (a + bi) and a real number (which can be regard
         <altmember cref="P:System.Numerics.Complex.Phase" />
       </Docs>
     </Member>
-    <MemberGroup MemberName="op_Multiply">
```

Comment:
This was supposed to be a MemberGroup for "Multiply", which is also a copy of the MemberGroup for "op_Multiply".
@rpetrusha Can you bring it back and leave the group, but with the MemberName fixed?